README.md

```diff
@@ -42,6 +42,22 @@ FULLNODE_API_INFO=/ip4/127.0.0.1/tcp/1234/http boost run
 boost dummydeal
 ```
 
+## Web UI
+
+1. Install and build
+
+```
+cd react
+npm install
+npm run build
```

Comment:
Is there also a `watch` for development mode? Would be great to have it documented if so.

Reply:
I added a "Development mode" section describing `npm start`